macosx/Controller.mm

```diff
@@ -884,10 +884,24 @@ - (void)applicationDidFinishLaunching:(NSNotification*)notification
                                                         andEventID:kAEOpenContents];
 
     //if we were opened from a user notification, do the corresponding action
-    NSUserNotification* launchNotification = notification.userInfo[NSApplicationLaunchUserNotificationKey];
-    if (launchNotification)
+    if (@available(macOS 10.14, *))
     {
-        [self userNotificationCenter:NSUserNotificationCenter.defaultUserNotificationCenter didActivateNotification:launchNotification];
+        UNNotificationResponse* launchNotification = notification.userInfo[NSApplicationLaunchUserNotificationKey];
+        if (launchNotification)
+        {
+            [self userNotificationCenter:UNUserNotificationCenter.currentNotificationCenter
+                didReceiveNotificationResponse:launchNotification withCompletionHandler:^{
```

Comment:
nit: can we pass `nil` here?

Reply:
no, the protocol doesn't accept a nullable handler.